lib/core/app/route_utils.dart

```diff
@@ -0,0 +1,49 @@
+enum Pages {
```

Comment:
what do you think about following improvement?

`enum Pages {
  splash(path: '/', name: 'SPLASH'),
  login(path: '/login', name: 'LOGIN'),
  profileSelection(path: '/profile-selection', name: 'PROFILE_SELECTION'),
  wallet(path: '/wallet', name: 'WALLET'),
  camera(path: '/camera', name: 'CAMERA'),
  success(path: '/success', name: 'SUCCESS'),
  chooseAmountSlider(
      path: '/choose-amount-slider', name: 'CHOOSE_AMOUNT_SLIDER');

  final String path;
  final String name;

  const Pages({
    required this.path,
    required this.name,
  });
}`

Reply:
agreed with @Daniela510 to add this suggestion by myself.